web-crypto/sign-verify/rsa-pss.js

```diff
@@ -0,0 +1,95 @@
+(function() {
+
+  /*
+  Store the calculated signature here, so we can verify it later.
+  */
+  let signature;
+
+  /*
+  Fetch the contents of the "message" textbox, and encode it
+  in a form we can use for sign operation.
+  */
+  function getMessageEncoding() {
+    const messageBox = document.querySelector(".rsa-pss #message");
+    let message = messageBox.value;
+    let enc = new TextEncoder();
+    return enc.encode(message);
+  }
+
+  /*
+  Get the encoded message-to-sign, sign it and display a representation
+  of the first part of it in the "signature" element.
+  */
+  async function signMessage(privateKey) {
+    const signatureValue = document.querySelector(".rsa-pss .signature-value");
+    signatureValue.classList.remove("valid");
+    signatureValue.classList.remove("invalid");
+
+    let encoded = getMessageEncoding();
+    signature = await window.crypto.subtle.sign(
+      {
+        name: "RSA-PSS",
+        saltLength: 32,
+      },
+      privateKey,
+      encoded
+    );
+
+    let buffer = new Uint8Array(signature, 0, 10);
+    signatureValue.textContent = buffer;
+  }
+
+  /*
+  Fetch the encoded message-to-sign and verify it against the stored signature.
+  * If it checks out, set the "valid" class on the signature.
+  * Otherwise set the "invalid" class.
+  */
+  async function verifyMessage(publicKey) {
+    const signatureValue = document.querySelector(".rsa-pss .signature-value");
+    signatureValue.classList.remove("valid");
+    signatureValue.classList.remove("invalid");
```

Comment:
`classList.remove()` should take multiple arguments, which cleans this up a little bit.